pgmpy/tests/test_models/test_DynamicBayesianNetwork.py

```diff
@@ -140,6 +140,43 @@ def test_moralize(self):
                               [('D', 1), ('G', 1)], [('D', 1), ('I', 1)],
                               [('G', 0), ('I', 0)], [('G', 1), ('I', 1)]])
 
+    def test_copy(self):
+        self.network.add_edges_from(
+            [(('D', 0), ('G', 0)), (('I', 0), ('G', 0)), (('D', 0), ('D', 1)), (('I', 0), ('I', 1))])
+        cpd = TabularCPD(('G', 0), 3, values=[[0.3, 0.05, 0.8, 0.5],
+                                                         [0.4, 0.25, 0.1, 0.3],
+                                                         [0.3, 0.7, 0.1, 0.2]],
```

Comment:
Run `PEP8` at here. I think this is over indented. You can use following command
 ```pep8 <file_name> --max-line-length=120
````